samples/Common.c

```diff
@@ -175,6 +179,35 @@ STATUS masterMessageReceived(UINT64 customData, PReceivedSignalingMessage pRecei
     return retStatus;
 }
 
+STATUS printSelectedIceCandidatesInformation(PSampleStreamingSession pSampleStreamingSession)
```

Comment:
I am so sorry, it seems. that we are using logBlahBlah. Do you mind making it consistent with the others and name this logSelectedIceCandidatesInformation? You can completely ignore it if you wish :)

Reply:
Makes sense. Fixed.